cmd/plasmacli/subcmd/keys/export.go

```diff
@@ -0,0 +1,47 @@
+package keys
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/store"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// ExportCmd returns the keys export command
+func ExportCmd() *cobra.Command {
+	return exportcmd
+}
+
+var exportcmd = &cobra.Command{
+	Use:   "export <name> <location>",
+	Short: "Export a private key",
+	Long: `Exports a private key to a specified location (must be absolute path).
```

Comment:
Is this true about the absolute path? I believe `os.OpenFile` will respect the cwd unless provided an absolute path

Reply:
You're right, good catch